### DISCUSSION TITLE
Support for "Named Arguments" in expression trees

### DISCUSSION THREAD

**Discussion by Der-Kraken** [dotnet/csharplang#5746] [Language Ideas] [answered]
Your colleague from efcore-repository proposed me that my issue belongs to c# repository. Here is my origin issue from ef-core dotnet/efcore#27413.

Like the [issue #28481](https://github.com/dotnet/roslyn/issues/28481) (optional parameters in expression trees) I suggesst to support "[Named Arguments](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/classes-and-structs/named-and-optional-arguments)" in expression trees too.

The reasons why you can read from the [issue #27413](https://github.com/dotnet/efcore/issues/27413) from efcore-repository.

**Comment by TahirAhmadov** [comment, marked as answer]
I second this, and in general, catching up on (almost) all new C# syntax since LINQ expressions were introduced. I believe there may be a thread about this somewhere.

**Reply by HaloFour** [reply]
Here it is, it's a bit of a catch-all: https://github.com/dotnet/csharplang/discussions/4727
